Actors/VisualObjects/uninstall.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
-sfctl application delete --application-id VisualObjectsApplication
-sfctl application unprovision --application-type-name VisualObjectsApplicationType --application-type-version 1.0.0
-sfctl store delete --content-path VisualObjectsApplication
\ No newline at end of file
+sfctl application delete --application-id VisualObjectsApplicationCSharp
+sfctl application unprovision --application-type-name VisualObjectsApplicationTypeCSharp --application-type-version 1.0.0
```

Comment:
The VO webservice, where does it get the application name?

Reply:
string appName = context.CodePackageActivationContext.ApplicationName; It is using this to get the application name